apcd-cms/src/client/src/utils/dateUtil.ts

```diff
@@ -13,3 +13,45 @@ export const formatDate = (dateString: string | number | Date): string => {
     hour12: true,
   }).format(date);
 };
+
+/**
+ *
+ * @param period string - Jan. 2024 as example
+ * @returns string. 2024-01 for Jan. 2024.
+ */
+export const convertPeriodLabelToApiValue = (period: string): string | null => {
```

Comment:
needed this because the api was returning Jan. 2024 which does not fit with db values